frontend/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@heroicons/react": "^2.1.1",
```

Comment:
Is this dependency being used? I could see it used anywhere.

Reply:
its not being used now i will remove it